atorch/examples/moe/train_moe_dummy_data.py

```diff
@@ -162,7 +182,8 @@ def forward(self, hidden_states):
         if self.shared_experts is not None and not self.use_expert_parallelism:
             hidden_states = hidden_states + self.shared_experts(identify)
 
-        return hidden_states
+        # Return the auxiliary loss along with the hidden states
+        return hidden_states, aux_loss
```

Comment:
This example is based on transformers llama model. 
llama mlp definition only returns hidden_states, return tuple won't work.